tide_webform.install

```diff
@@ -220,3 +220,34 @@ function tide_webform_update_8007() {
     $block->save();
   }
 }
+
+/**
+ * Updating service version.
+ */
+function tide_webform_update_8008() {
+  if (\Drupal::moduleHandler()->moduleExists('tide_webform')) {
```

Comment:
hey  @edyuenyw  doesn't need to check whether tide_webform exists or not.

Reply:
Thank you @vincent-gao. Will remove it.